tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -95,6 +99,29 @@ export const acceptShare = async (args: ShareStatusArgs): Promise<void> => {
   await page.locator(util.format(filesSharedWithMeAccepted, resource)).waitFor()
 }
 
+export const acceptAllShare = async ({ page }: { page: Page }): Promise<void> => {
+  if (await page.locator(showAllButton).isVisible()) {
+    await page.locator(showAllButton).click()
+  }
+  await page.locator(selecAllCheckbox).click()
+  const declinedShareItems =
+    '//div[@id="files-shared-with-me-pending-section"]//tr[contains(@class,"oc-tbody-tr")]'
```

Comment:
you can move it to the list of selectors above.

Reply:
done